src/box/lua/call.c

```diff
@@ -673,6 +721,7 @@ static const struct port_vtab port_lua_vtab = {
 	.dump_plain = port_lua_dump_plain,
 	.get_msgpack = port_lua_get_msgpack,
 	.get_vdbemem = port_lua_get_vdbemem,
+	.enlight = port_lua_enlight,
```

Comment:
I'd prefer to have `get_c_entry` callback that would return one (or more) entries in the C format from any port.

Reply:
Done.